### PR TITLE
HDDGURU.HDDRawCopyTool version 1.10

### PR DESCRIPTION
**Pull request by Masamune3210**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

**Comment by Masamune3210** [comment]
I'm not sure if I'm doing something wrong but I cant get winget to use any local manifests, if I try to validate it complains that the multi-file manifest is incomplete, and if I try to install it says that the manifest could not be opened

File: manifests/h/HDDGURU/HDDRawCopyTool/1.10/HDDGURU.HDDRawCopyTool.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: HDDGURU.HDDRawCopyTool
+PackageVersion: 1.10
+PackageLocale: en-US
+Publisher: HDDGURU
+PackageName: HDD Raw Copy Tool
+License: Copyright 2005-2021 HDDGURU.COM
+ShortDescription: HDD Raw Copy Tool Setup
```
Comment (ItzLevvie) [inline manifests/h/HDDGURU/HDDRawCopyTool/1.10/HDDGURU.HDDRawCopyTool.locale.en-US.yaml:10]:
```
ShortDescription: HDD Raw Copy Tool Setup
```
Please write a better description about what this application is about.

File: manifests/h/HDDGURU/HDDRawCopyTool/1.10/HDDGURU.HDDRawCopyTool.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: HDDGURU.HDDRawCopyTool
+PackageVersion: 1.10
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://hddguru.com/software/HDD-Raw-Copy-Tool/HDDRawCopy1.10Setup.exe
+  InstallerSha256: BA53F3B60933CBFABD74F6BCBE68677C15ED18FFE683F4BAAF9F4AA2E9D1069B
```
Comment (ItzLevvie) [inline manifests/h/HDDGURU/HDDRawCopyTool/1.10/HDDGURU.HDDRawCopyTool.installer.yaml:10]:
```suggestion
  InstallerSha256: BA53F3B60933CBFABD74F6BCBE68677C15ED18FFE683F4BAAF9F4AA2E9D1069B
  ProductCode: HDD Raw Copy Tool_is1
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
> I'm not sure if I'm doing something wrong but I cant get winget to use any local manifests, if I try to validate it complains that the multi-file manifest is incomplete, and if I try to install it says that the manifest could not be opened

Are you sure you did `winget install --manifest "manifests/h/HDDGURU/HDDRawCopyTool/1.10"`?

**Comment by Masamune3210** [comment]
> > I'm not sure if I'm doing something wrong but I cant get winget to use any local manifests, if I try to validate it complains that the multi-file manifest is incomplete, and if I try to install it says that the manifest could not be opened
> 
> Are you sure you did `winget install --manifest "manifests/h/HDDGURU/HDDRawCopyTool/1.10"`?

I just did now and it worked fine, my issue was I was pointing it to a specific file, was unaware it was supposed to be the entire folder, thank you

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**